src/utils/combineRegExps.js

```diff
@@ -0,0 +1,23 @@
+const combineRegExps = regExps => {
+  const serializedRegexParser = /^\/(.+)\/(\w+)?$/
+  return new RegExp(
+    regExps
+      .map(regex => {
+        const [, regexString, regexFlags] = serializedRegexParser.exec(
+          regex.toString()
+        )
+
+        if (regexFlags) {
+          throw new Error(
```

Comment:
Do we include the `invariant` package? I kinda like how invariants don't force me to think so much in `if-else` terms